jekyll/_cci2/configuration-reference.md

```diff
@@ -27,12 +27,62 @@ You can see a complete `config.yml` in our [full example](#full-example).
 
 Key | Required | Type | Description
 ----|-----------|------|------------
-version | Y | String | `2`, `2.0`, or `2.1` See the Reusing Config doc for an overview of new 2.1 keys available to simplify your `.circleci/config.yml` file, reuse, and parameterized jobs.
+version | Y | String | `2`, `2.0`, or `2.1` See the [Reusing Config]({{ site.baseurl }}/2.0/reusing-config/) doc for an overview of new 2.1 keys available to simplify your `.circleci/config.yml` file, reuse, and parameterized jobs.
 {: class="table table-striped"}
 
 The `version` field is intended to be used in order to issue warnings for deprecation or breaking changes.
 
-## **`commands`** (requires version:2.1)
+## **`orbs`** (requires version: 2.1)
+
+Key | Required | Type | Description
+----|-----------|------|------------
+orbs | N | String | The name of an orb to use.
+executors | N | String | The name of an executor to declare.
+commands | N | String | The name of a command to declare.
+jobs | N | String | The name of a job to declare.
+{: class="table table-striped"}
+
+The following example calls an Orb named `hello-build` that exists in the certified `circleci` namespace.
+
+```
+version: 2.1
+orbs:
+    hello: circleci/hello-build@volatile
+workflows:
+    "Hello Workflow":
+        jobs:
+          - hello/hello-build
+```
+
+The following example declares an inline Orb in the `config.yml` file for the project.
```

Comment:
See above - I think we should keep inline orbs in their own page, as it's a fairly specialized thing to want to do related to the orb authoring lifecycle, so it's not really something people building general config should have to think about. We can then link to that page in discussion of the `orbs` stanza rather than documenting it here.

Reply:
One thing that came up from Support (that I had issues with as well), is that when folks start with an inline orb and then shift to using it as a sample orb, they'll often leave the top-level `orbs` key, which creates a funky broken state for them.

In effect, they'll wind up publishing some orb like `hannah/foo@dev:bar`:
```
orbs:           # This whole config becomes the orb `bar`
    bar:         # This inline `bar` doesn't exist in the published orb `bar`
        jobs:
            baz-job: ...  # When users import this config as the orb `bar`, and think they're calling
                                # `bar/baz-job`, they're trying to access `bar/bar/baz-job`, and we don't
                                # support references to nested inline orbs
```

Reply:
thanks, removed